Makefile

```
@@ -1,6 +1,10 @@
 build:
 	@yarn turbo run build
 	@node packages/compiled-assets/dist/cli.js build ./assets ./public/build
+deps:
+	@yarn
+	@pip3 install -qr images/plbase/python-requirements.txt --root-user-action=ignore
```

Open to discuss this, but I generally like confirmation that a command is doing something. It took me a while to figure out that `make deps` wasn't actually broken when I didn't see output from pip.

Also changing this to use `python3 -m pip` instead of `pip3`, as this is what we use in the Markdown docs.

```suggestion
	@python3 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
```

---

I had originally suggested `-q` for a command that would run in the background, but it makes sense to remove it here. As for `python3 -m`, I was checking online and there are minor advantages to the latter, but most don't apply to our scenario, such as controlling the version of `python3` in use (we typically only have one) or replacing the executable in Windows. Regardless, I'm changing to that in `accb07489` for consistency with best practices. I'm also creating a separate `make python-deps` to avoid having the full command on docs.